### ISSUE TITLE
Equivalence of torch.multinomial in ACL

### ISSUE THREAD

**Issue by kumarchaitanyab** [ARM-software/ComputeLibrary#1166]
Team can you please point me to an Equivalence Torch function torch.multinomial using ACL library? So far I coule not find it.

**Comment by morgolock** [comment]
Hi @kumarchaitanyab 

I'd like to know more about your requirements and use case. Please share more details that would help us to understand how you're using ACL.

We don't have a direct equivalent to PyTorch's `torch.multinomial` in the library, this is because ACL is designed for neural network inference. We have most common inference operators like gemm, activations, normalization. PyToch's  `torch.multinomial` is commonly used for training and this is out of the scope of ACL.

You could explore implementing your own operator in ACL for this. We would be happy to review your patch and merge it to main.

Hope this helps,
